web/src/components/troubleshoot/PreflightsProgress.tsx

```diff
@@ -1,25 +1,42 @@
 import React from "react";
 import Loader from "../shared/Loader";
 import "@src/scss/components/AirgapUploadProgress.scss";
-import { getReadableCollectorName } from "../../utilities/utilities";
 
-let percentage;
-
-function moveBar(count) {
+function moveBar(percentage: number) {
   const elem = document.getElementById("preflightStatusBar");
-  percentage = count > 21 ? 96 : (count * 4.5).toFixed();
   if (elem) {
-    elem.style.width = percentage + "%";
+    elem.style.width = percentage.toFixed() + "%";
+  }
+}
+
+interface NamesObj {
+  [key: string]: string;
+}
+function getReadableCollectorName(name: string) {
+  const namesObj: NamesObj = {
+    "cluster-info": "Gathering basic information about the cluster",
+    "cluster-resources": "Gathering available resources in cluster",
+    mysql: "Gathering information about MySQL",
+    postgres: "Gathering information about PostgreSQL",
+    redis: "Gathering information about Redis",
+  };
+  if (name in namesObj) {
+    return namesObj[name];
   }
+
+  return "Gathering details about the cluster";
 }
```

Comment:
this was in the utilities.js but it's only used here so I moved it out of there to reduce indirection